pkgdb/Makefile

```diff
@@ -182,6 +182,7 @@ HEADERS        = $(call rwildcard,include,*.hh)
 SRCS           = $(call rwildcard,src,*.cc)
 test_SRCS      = $(sort $(wildcard tests/*.cc))
 ALL_SRCS       = $(SRCS) $(test_SRCS)
+ASSETS         = $(shell find src/buildenv/assets -type f)
```

Comment:
@ysndr add a comment that the build uses a nix store path and pulling in assets in the makefile is mainly for dev purposes (to be verified)